apps/01PigLatin.js

```diff
@@ -64,3 +120,4 @@ if (typeof describe !== 'undefined') {
     getPrompt();
 
 }
```

Comment:
Good job on the app, but I am not sure based off your answers that you completely understand how the app is working. Read some of the links to the MDN documentation that is provided in the comments of this code and try to fix you answers at a later time. I will give you full credit for this assignment for now.